src/cascadia/TerminalApp/Pane.h

```diff
@@ -19,140 +19,169 @@
 // - Mike Griese (zadjii-msft) 16-May-2019
 
 #pragma once
-#include <winrt/Microsoft.Terminal.TerminalControl.h>
+#include <functional>
 #include <winrt/TerminalApp.h>
 #include "../../cascadia/inc/cppwinrt_utils.h"
 
-enum class Borders : int
-{
-    None = 0x0,
-    Top = 0x1,
-    Bottom = 0x2,
-    Left = 0x4,
-    Right = 0x8
-};
-DEFINE_ENUM_FLAG_OPERATORS(Borders);
+class LeafPane;
+class ParentPane;
 
-class Pane : public std::enable_shared_from_this<Pane>
+class Pane
 {
 public:
-    Pane(const GUID& profile,
-         const winrt::Microsoft::Terminal::TerminalControl::TermControl& control,
-         const bool lastFocused = false);
+    virtual ~Pane() = default;

```

Comment:
Rule of 5. If defining a destructor, also define the copy/move constructor and copy/move operator=.